fsync.go

```diff
@@ -85,7 +86,7 @@ func (s *Syncer) Sync(dst, src string) error {
 // SyncTo syncs srcs files or directories into to directory.
 func (s *Syncer) SyncTo(to string, srcs ...string) error {
 	for _, src := range srcs {
-		dst := path.Join(to, path.Base(src))
```

Comment:
Oh.. good call. It probably should, but it worked fine for me without doing that.